__tests__/index.js

```diff
@@ -2,6 +2,7 @@ const find = require('../src/index')
 var cp = require('mz/child_process')
 
 const TEN_MEGA_BYTE = 1024 * 1024 * 10
+const MAX_ARP_TIME = 60 * 60 * 10
```

Comment:
```suggestion
const TIMEOUT_MS = 60 * 1000
```

I would suggest renaming the variable and increase the timeout to 60s or decrease to 30s. And update in the other files as well, please.

Reply:
Totally agree 👍

Reply:
This is not a big deal, but I’d personally like to see the variable called `ONE_MINUTE`, to make it clear what the intent is without having to do the (fairly simple) calculation yourself. It also better aligns with the `TEN_MEGA_BYTE` const we already have.

Reply:
I have changed the variable name